### ISSUE TITLE
it seems for various users the browser freezes when editing the gallery

### ISSUE THREAD

**Issue by mplusb** [WPChill/modula-lite#540]
**Describe the bug**
It seems to happen randomly: 
When I go to edit a gallery and click on "captions" for example, I get a yellow bar in the browser window saying a script is slowing down the page. The Captions tab never loads. Some galleries have the problem, while other galleries work fine. 

<img width="1339" alt="Screenshot 2021-03-25 at 09 26 38" src="https://user-images.githubusercontent.com/26008574/112434678-85fe5280-8d4c-11eb-8762-d2f99a222978.png">

All plugins disabled except Modula and Modula Pro
On my Windows 10 computer on Firefox, i can edit the Modula galleries with no problem.
On Name1's Linux computer in Firefox / Chromium / Brave the browser freezes up when he tries to edit a gallery.
On a Windows 7 VM on Name1's computer, the same thing happens, the browser freezes up.

I opened a ticket with Dreamhost support to see if there were any potential problems. Here is their response. I disabled mod_security per their suggestion and Name1 reports his browser still freezes up when he tries to edit a Modula gallery.

Sorry to read the trouble with accessing your site on your Linux machine
and on specific browsers. Thanks for sharing the details and for
isolating where the error is occurring. I did find mod_security entries
in the error.log but not sure if that entirely sure if that is the cause
of the issue:

[Sat Mar 20 10:01:22.455697 2021] 
ModSecurity:
Warning. Matched phrase "/wp-admin/edit.php" at REQUEST_HEADERS:Referer.
[file
"/mod_sec3_CRS/REQUEST-00-LOCAL-WHITELIST.conf"]
"/wp-admin/admin-ajax.php"] 
referer:
/edit.php?post_type=modula-gallery

You can temporarily disable mod_security to further isolate if the
firewall is blocking the connection from your Linux host. The details and
steps to enable/disable mod_security can be referenced here -
https://help.dreamhost.com/hc/en-us/articles/215947927

When disabling mod_security, please allow at least 5-7 mins for the
changes to apply and then kindly retry the steps to confirm the pages are
accessible on your Linux host. If the issue with editing your galleries
still occurs, please do share the timestamp and URL of when the error
occurs and we can take a further look at the error.log. 

**To Reproduce**
was unable to reproduce on my end, but it happened a few other users.

https://secure.helpscout.net/conversation/1382001075/34150/

**Comment by mplusb** [comment]
https://secure.helpscout.net/conversation/1420755644/35413

**Comment by mplusb** [comment]
2 more: https://secure.helpscout.net/conversation/1469078960/36828?folderId=1289779
https://secure.helpscout.net/conversation/1471577395/36891?folderId=1289779

**Comment by mplusb** [comment]
2 more users: 1 has IIS server
1 has: Apache running on Ubuntu Linux through DreamHost.com

**Comment by mplusb** [comment]
found the issue:

In order to help you a little bit I have installed locally a fresh wordpress version of 5.7 without any further plugins. Installed and activated Modula v2.5. Then I added a new gallery with 3 sample images and as long I keep the Gallery bar open everything works fine. But if I close the Gallery bar and try to change something e.g. the Gallery Type from Creative Gallery to Custom Grid or try to Update Gallery the browser stucks.

![PastedGraphic-3](https://user-images.githubusercontent.com/26008574/113678021-dc617e80-96c6-11eb-96ed-4bc4fc691fc2.png)
![PastedGraphic-4](https://user-images.githubusercontent.com/26008574/113678029-de2b4200-96c6-11eb-9462-ec1ef59b7ee8.png)
